components/notifier.rst

```diff
@@ -0,0 +1,174 @@
+.. index::
+   single: Notifier
+   single: Components; Notifier
+
+The Notifier Component
+====================
+
+    The Notifier component helps sending emails.
+
+If you're using the Symfony Framework, read the
+:doc:`Symfony Framework Notifier documentation </notifier>`.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/notifier
+
+.. include:: /components/require_autoload.rst.inc
+
+Email
+-----
+
+The Notifier component has notify you when something goes wrong::
```

Comment:
```suggestion
The Notifier component can notify someone(s) when something occurs::
```

it can be you or someone else, that the devops or whomever
also, not only when goes wrong, it can be something good happening :)